microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -70,164 +71,274 @@ After doing the above steps, the extension is ready to be submitted via Partner
+| Property name | Required? | Description |
+|---|---|---|
+| **Category** | Required | The category that best describes your extension.  Listing your extension in the right category helps users find your extension easily and understand more about it.  |
+| **Privacy policy requirements** | Required | Indicate if your extension accesses, collects, or transmits any personal information.  Your extension might fail the certification step if you select **Yes** and you don't provide a `Privacy policy URL`. |
+| **Privacy policy URL** | - | A valid privacy policy URL to communicate how your extension follows privacy laws and regulations.  You're responsible for ensuring your extension follows privacy laws and regulations.  You're also responsible for providing a privacy policy URL if any personal information is being accessed, transmitted, or collected by your extension.  To determine if your extension requires a privacy policy, see [Microsoft Store App Developer Agreement](https://go.microsoft.com/fwlink/?linkid=528905) and [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
+| **Website URL** | - | A webpage that provides additional information about your extension.  The `Website URL` must point to a webpage on your own website, not the web listing for your extension in the [Microsoft Edge Add-ons website](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home).  The `Website URL` helps users learn more about your extension, its features, and any other relevant information. |
+| **Support contact details** | - | The URL to your support webpage, or the email address to contact your support team. |
+| **Mature content** | - | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, see [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
 
-Select **Save & Continue** to continue to the **Store listings** section.
+Properties that are not indicated as Required are Optional.
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Your developer/organization name, **Website URL**, and **Support contact details** that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+
+After filling in the **Properties** page for the extension, click the **Save & Continue** button.
+
+The **Store listings** page opens.
```

Comment:
```suggestion
The **Store Listings** page opens.
```
@captainbrosset - this is the extra one like one you pointed out.
@bhuvanapriyap - at Partner Ctr, left nav says "listings", top of page title says "Listings".  We're preferring the top-of-page capzn over mere nav capzn, when not specifically saying to click the nav text.